src/flightlog_parser.js

```
@@ -776,208 +915,671 @@ export function FlightLogParser(logData) {
+          (this.sysConfig.firmwareType == FIRMWARE_TYPE_CLEANFLIGHT &&
+            semver.gte(this.sysConfig.firmwareVersion, '2.0.0'))
+        ) {
+          this.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue)
+        } else {
+          this.sysConfig[fieldName] = parseInt(fieldValue, 10) / 100.0
+        }
+        break
+
+      case 'digitalIdleOffset':
+        this.sysConfig[fieldName] = parseInt(fieldValue, 10) / 100.0
+
```

ESLint

[error] Missing semicolon.

(semi)